components/ui/AppPageHeaderWithCard.vue

```diff
@@ -1,22 +1,26 @@
 <template>
   <header class="app-page-header">
     <div class="app-page-header__container">
-      <main class="app-page-header__main">
+      <div class="app-page-header__main">
```

Comment:
According to the HTML specification, `main` must be "[hierarchically correct](https://html.spec.whatwg.org/#hierarchically-correct-main-element)" and so:
> A hierarchically correct main element is one whose ancestor elements are limited to html, body, div, form without an accessible name, and autonomous custom elements. Each main element must be a hierarchically correct main element.

Reply:
@y4izus @eddybrando ⬆️